app/controllers/ncr/work_orders_controller.rb

```diff
@@ -14,7 +14,9 @@ def create
     end
 
     def edit
-      if self.proposal.approved?
+       @client_data_instance.approving_official_email = @client_data_instance.approvers.first.email_address
```

Comment:
the reason I didn't do that is that the value of the normalizer is taking raw values and converting them to formatted values -- like formatting strings etc. So running that whole class feels like overkill when all we need to do it set the one attribute.

I also feel like the `approving_official_email` stuff in general is a little wonky. Wish we would remove it altogether and just treat it like an association. I am sure there are good reason that it is the way that it is today, but in the future I'd love to see how we could not need to set that attribute manually and just let the form handle it.